submissions/kolawoleOgunfowokan/PHPTask/php/fetch.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require "config.php";
+/**
+ * This file Handles Php Data Object connection to Db
+ * which can be of any type e.g SQL db and runs commands in init.sl
+ */
+
+try {
+    session_start();
+    $connection = new PDO("mysql:host=$host", $username, $password, $options);
+    $stmt = $connection->query("SELECT * FROM myoja.brands");
+    $brands = array();
+    while ($item = $stmt->fetch()) {
+        array_push($brands, $item['brand_name']);
```

Comment:
@kolawolejohn, map your brand_id to brand_name here, so you won't have to do +1 while saving the product.
You can replace array_push (line 14) with something like `$brands[$item['brand_name']] =  $item['brand_name'];`. 
This way you are assured that the each brand carries it's id in the database.
Your current approach will not work if your brand ids are not in order in the database.
 I will add the remaining comments below:

Reply:
I am through with this and the necessary changes effected. One long ride init. "Foreach" saved a life.